guides/user/applications/configure.md

```diff
@@ -0,0 +1,144 @@
+---
+layout: single
+title:  "Configure an application"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+Once you've [created an application](/guides/user/applications/create/), you can
+finish configuring it before adding infrastructure and pipelines.
+
+## Start configuring
+
+1. From within the application, click **CONFIG** at the top right of the
+Spinnaker screen.
+
+   The application attributes you [defined when you created the
+   app](/guides/user/applications/create/) are shown at the top.
+
+   > **Note** Spinnaker can infer applications based on existing infrastructure.
+   > In these cases, there are no application attributes, and you'll have to
+   > add them.
```

Comment:
When you say Spinnaker infers applications based on existing infrastructure, do you mean the applications it automatically creates for Kubernetes providers? If so, do we actually want to encourage users to add application attributes to them?

Reply:
It depends on if they want application attributes for them. An example, if they are indexing `kube-system` (I still have to add the PR to call that out), and things like `heapster` that shouldn't be touched by Spinnaker show up they probably don't want to add application attributes. On the other hand, if they are already managing a large # of applications & want Spinnaker to pick them up, they should add application attributes.

Reply:
I changed the wording somewhat. PTAL, @nhayes and @lwander

Reply:
LGTM